docs/general/profiles.md

```diff
@@ -0,0 +1,55 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Logs
+aliases: [docs/specs/semconv/general/profiles-general]
+--->
+
+# General Profiles Attributes
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- toc -->
+
+- [Frame types](#frame-types)
+
+<!-- tocstop -->
+
+The attributes described in this section are rather generic.
+They may be used in any Profiles record they apply to.
+
+## Frame types
+
+**Description:** Describes the origin of a single frame in a Profile.
+
+<!-- semconv profile.frame(full) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`profile.frame.type`](/docs/attributes-registry/profile.md) | string | Describes the interpreter or compiler of a single frame. | `python` | `Opt-In` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`profile.frame.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `dotnet` | [.NET](https://wikipedia.org/wiki/.NET) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `jvm` | [JVM](https://wikipedia.org/wiki/Java_virtual_machine) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `kernel` | [Kernel](https://wikipedia.org/wiki/Kernel_(operating_system)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `native` | [C](https://wikipedia.org/wiki/C_(programming_language)), [C++](https://wikipedia.org/wiki/C%2B%2B), [Go](https://wikipedia.org/wiki/Go_(programming_language)), [Rust](https://wikipedia.org/wiki/Rust_(programming_language)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `perl` | [Perl](https://wikipedia.org/wiki/Perl) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `php` | [PHP](https://wikipedia.org/wiki/PHP) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `python` | [Python](https://wikipedia.org/wiki/Python_(programming_language)) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
You need to re-generate the table with `make fix` after changing to `cpython`

Reply:
@lmolkova shouldn't some check fail due to outdated markdown? The id of one of the enum names changed, but it is not reflected in the markdown here 🤔

Reply:
Did run `make fix` which fixed the name issue - see https://github.com/open-telemetry/semantic-conventions/pull/1188/commits/b6aed34588cc8313eb789884f87d7acc54a3f7c4. Not sure, why some commands did generate the correct output, like in `docs/attributes-registry/profile.md`, but this file did not get updated.

Reply:
Most likely you ran only the command to update the registry. They are separated and `make fix` executes both :)